install.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

To make it more robust.